all-rit/ALL
# 583 story import newsletter email component

To test:
Go to `http://localhost:3000/Lab12/Exercise/AlumniNewsletter` and ensure that you see the name 'Test' in the letter address and 'Dear Test'

In Lab 12's `Main.js`, change the name prop and ensure the name change is reflected.

In Lab 12's constant `index.js`, change the content of the `ALUMNI_NEWSLETTER_CONTENT` variable and ensure the change is reflected.

- [ x] No discrepancies across browsers. Eg. chrome vs safari
- [ x] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ x] CSS (https://jigsaw.w3.org/css-validator/)
- [ x] Google sign-in is operational
- [ x] Accessibility functions
- [ x] Pages can scale without distorting page
- [x ] No dead links
- [ x] Navbar is consistent across the site
- [ x] Pages are screen-reader accessible
- [ x] Contrast meets standards for accessibility
- [ x] Pages are keyboard accessible
- [ x] Passes WAVE Evaluation Tool chrome extension
